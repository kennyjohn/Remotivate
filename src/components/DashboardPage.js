import React from 'react';
import SelectEntryType from './SelectEntryType';
import EntryList from './EntryList';
import Spotlight from './Spotlight';

const DashboardPage = () => (
  <div>
    <Spotlight></Spotlight>
    <SelectEntryType></SelectEntryType>
    <EntryList></EntryList>
  </div>
);

export default DashboardPage;
