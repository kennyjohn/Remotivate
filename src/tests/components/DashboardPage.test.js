import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/DashboardPage';

test('should render DashboardPage', () => {
    const wrapper = shallow(<DashboardPage></DashboardPage>)
    expect(wrapper).toMatchSnapshot();
});
