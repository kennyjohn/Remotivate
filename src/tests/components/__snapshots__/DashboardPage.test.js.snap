// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage 1`] = `
<div>
  <Spotlight />
  <SelectEntryType />
  <EntryList />
</div>
`;
